RELEASE.md

```diff
@@ -10,7 +10,7 @@ To cut a pre-release:
 
 To cut a release:
 1. Ensure the relevant spec APIs have been released.
-1. Ensure any [migration guides](https://github.com/buildpacks/docs/tree/main/content/docs/reference/spec/migration) have been created.
+1. Ensure the `lifecycle/0.99.0` milestone on the [docs repo](https://github.com/buildpacks/docs/blob/main/RELEASE.md#lump-changes) and associated branch are complete / ready to merge, including [migration guides](https://github.com/buildpacks/docs/tree/main/content/docs/reference/spec/migration) (if relevant).
```

Comment:
Isn't it confusing to write "ready to merge"?
What does it really mean?
Maybe we should be more specific?

Reply:
Please see 88c2200